recipe/meta.yaml

```diff
@@ -22,8 +22,8 @@ requirements:
     - wheel
     - setuptools
   run:
-    - protobuf >=3.12.0
-    - python
+    - protobuf >=3.19.0
```

Comment:
Do we really need >= 3.19.0 or will an older version do, as well? protobuf on win-32 cannot be upgraded anymore, so with this constraint this proto-plus upgrade wouldn't make it there either. The latest available version on win-32 is 3.14 but the pinning (of libprotobuf) is at 3.11, iinm. Does this build against 3.11?

Reply:
Oh, this is noarch...never mind.

Reply:
Unfortunately it does seem that older version recommended for that version is `protobuf` version `3.19.0` or higher.

https://github.com/googleapis/proto-plus-python/blob/main/setup.py#L39